lms/djangoapps/dashboard/decisions/0002-deprecate-sysadmin-dashboard-adr

```diff
@@ -0,0 +1,45 @@
+1. Deprecating the Sysadmin Dashboard
+---------------------
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+Maintaining the sysadmin dashboard is challenging, and it is not widely used. The code is part of the lms
+application, even though most of it's use cases are relevant to course authoring.
+
+The sysadmin dashboard would be better suited as a pluggable django application, using appropriate APIs in the
+cms application
+
+Decision
+--------
+
+In order to deprecate the sysadmin dashboard and move it to a pluggable django application, the followings APIs
+would need to be added and/or moved into the cms application
+
+1. Create a cms user account
+
+  https://github.com/mitodl/edx-platform/blob/50dd1238408dc6785f022d8540961f96e0d6bb4f/lms/djangoapps/dashboard/sysadmin.py#L113-L151
+
+2. Delete a cms user.
+
+  This functionality should be removed entirely. CMS user accounts should be retired using the procedure described
+  ... instead of simply deleting them.
```

Comment:
Needs a link to user retirement documentation

Reply:
User Retirement developer doc: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/user_retire/

User Retirement user doc: https://support.edx.org/hc/en-us/articles/206915007-How-can-I-delete-my-account-

Reply:
Thanks, will add a link to the developer doc.